packages/launcher/test/unit/detect_spec.ts

```diff
@@ -110,7 +145,7 @@ describe('browser detection', () => {
             info: 'Loaded from /foo/bar/browser',
             custom: true,
             version: '9001.1.2.3',
-            majorVersion: 9001,
+            majorVersion: '9001',
```

Comment:
We were previously setting the majorVersion as an integer version if it could be parsed in [`setMajorVersion`](https://github.com/cypress-io/cypress/pull/23164/files#diff-3c69c99db158ca335f24134e072fa8e4b28c55cec925262ce1959aca87c986aaL30-L34). But it's defined as a string on the FoundBrowser type and looks like it should to be (sometimes the version contains chars). So I normalized the type being set, and updated these tests.